index.js

```diff
@@ -10,9 +10,39 @@ function parseGitUri(uri) {
     uri.match(/git@github.com:(.*)\/(.*)\.git/);
 }
 
+function verifyChangelog() {
+  var changelog = exec('git diff HEAD~1');
+  var lines = changelog.split('\n');
+  var isOneLine = (lines.length === 1);
+  var deletions = 0;
+  var containsErrorMessage = false;
+  lines.filter(function (line) {
+    if (/^-/.test(line)) deletions++;
+    if (line.indexOf('Make a POST first') >= 0) containsErrorMessage = true;
+  });
+  if (isOneLine) {
+    revertChanges();
+    console.error('Changelog diff should be more than 1 line long');
+    process.exit(3);
+  }
+  if (deletions > 10) {
```

Comment:
Ok. If 2 is too strict, we can increase this later. @freitagbr can you move the number into a variable at the top and add an explanatory comment (briefly explain which two lines typically get changed).